app/components/UI/TooltipButton.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box, Text } from 'rebass'
+
+const StyledTooltipIconBox = styled(Box)`
+  border-radius: 50%;
+  height: 16px;
+  line-height: 16px;
+  margin-left: 5px;
+  position: relative;
+  text-align: center;
+  transform: translateZ(0);
+  width: 16px;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+  }
+`
+StyledTooltipIconBox.displayName = 'TooltipIconBox'
+
+const StyledTooltipWrapper = styled(Box)`
+  border: 1px solid white;
+  border-radius: 4px;
+  padding: 5px 5px 5px 25px;
+  position: absolute;
+  top: -5px;
+  width: 225px;
+  z-index: 1;
+  transform: translateZ(0);
+`
+StyledTooltipWrapper.displayName = 'TooltipWrapper'
+
+/**
+ * @render react
+ * @name TooltipButton
+ * @example
+ * <TooltipButton>Some explanation text here</TooltipButton>
+ */
+class TooltipButton extends React.Component {
+  static displayName = 'TooltipButton'
+
+  state = { hover: false }
+
+  hoverOn = () => {
+    this.setState({ hover: true })
+  }
+
+  hoverOff = () => {
+    this.setState({ hover: false })
+  }
+
+  render() {
+    const { children, ...rest } = this.props
+    const { hover } = this.state
+
+    return (
+      <Box
+        style={{
+          position: 'relative'
+        }}
+      >
+        <StyledTooltipIconBox
+          bg="white"
+          color="primaryColor"
+          onMouseEnter={this.hoverOn}
+          onMouseLeave={this.hoverOff}
+          {...rest}
+        >
+          <Text fontWeight="bold" fontFamily="sans">
+            ?
+          </Text>
+        </StyledTooltipIconBox>
+        {hover ? (
+          <StyledTooltipWrapper bg="secondaryColor">
+            <Text fontWeight="normal" fontFamily="sans" lineHeight="1.4">
```

Comment:
`lineHeight="1.4"` why use custom line height here ? We have a clear style guidelines which are achieved with a set of defaults in most of the components. Please see `npm run storybook` for the reference.  One shouldn't use custom styles unless absolutely necessary

Reply:
The default line height seemed too small and didn't look like the mock-up.  I will adjust it to match the style guide.

Reply:
I'm looking at the style guide and can't find anything about line height there. Also searched the source for `lineHeight` and `line-height` to see if there are any line height global variables defined but didn't find anything. Am I missing something here?

Reply:
What I meant to say is that `Text` component already has a proper line height, so usually no need to set it manually

Reply:
Ok, I will just remove it then.